packages/http/src/models/Sum.ts

```diff
@@ -21,30 +22,65 @@ function floatToBalance(floatValue: number): Balance<NetworkTokens> {
 export default class Sum extends DataModel {
   private client: Client
 
-  public total: Balance<NetworkTokens>
+  public total: number
 
-  public stddev: Balance<NetworkTokens>
+  public balanceTotal: Balance<NetworkTokens>
 
-  public min: Balance<NetworkTokens>
+  public stddev: number
 
-  public median: Balance<NetworkTokens>
+  public balanceStddev: Balance<NetworkTokens>
 
-  public max: Balance<NetworkTokens>
+  public min: number
 
-  public avg: Balance<NetworkTokens>
+  public balanceMin: Balance<NetworkTokens>
 
-  constructor(client: Client, rewards: HTTPSum) {
+  public median: number
+
+  public balanceMedian: Balance<NetworkTokens>
+
+  public max: number
+
+  public balanceMax: Balance<NetworkTokens>
+
+  public avg: number
+
+  public balanceAvg: Balance<NetworkTokens>
+
+  public sum: number
+
+  public balanceSum: Balance<NetworkTokens>
+
+  public type: SumsType
+
+  constructor(client: Client, rewards: HTTPSum, type: SumsType) {
```

Comment:
does `rewards` make sense here as a variable name?

Reply:
Good point, changed it to `data`